ci/doc-index.sh

```diff
@@ -1,18 +1,25 @@
 #!/usr/bin/env bash
+
 set -e
+set -u
 
-BUILD_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-source ${BUILD_DIR}/ci/common/common.sh
-source ${BUILD_DIR}/ci/common/doc.sh
+readonly BUILD_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+readonly DOC_INDEX_PAGE_URL="${DOC_INDEX_PAGE_URL:-https://neovim.io/doc_index}"
+readonly DOC_SUBTREE='index.html'
 
-DOC_INDEX_PAGE_URL=${DOC_INDEX_PAGE_URL:-https://neovim.io/doc_index}
+source "${BUILD_DIR}/ci/common/common.sh"
+source "${BUILD_DIR}/ci/common/doc.sh"
 
 generate_doc_index() {
-  echo "Updating index.html from ${DOC_INDEX_PAGE_URL}."
-  curl -L --tlsv1 ${DOC_INDEX_PAGE_URL} > ${DOC_DIR}/index.html
+  echo "Updating index.html from: ${DOC_INDEX_PAGE_URL}"
+  # TODO: Can the 'tlsv1' flag be removed?
```

Comment:
see 0e9f559802ccc53530a4b2d3afab703f49df8e19

So, don't need that comment. If it is proven that it isn't needed we can remove it later.